scaling_performance/scaling_cluster_metrics.adoc

```diff
@@ -0,0 +1,85 @@
+[[scaling-performance-cluster-metrics]]
+= Scaling Cluster Metrics
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+
+== Overview
+
+{product-title} exposes metrics that can be collected and stored in back-ends by
+link:https://github.com/GoogleCloudPlatform/heapster[Heapster]. As an
+{product-title} administrator, you can view containers and components metrics in
+one user interface. These metrics are also used by
+xref:../dev_guide/pod_autoscaling.adoc#dev-guide-pod-autoscaling[horizontal pod
+autoscalers] in order to determine when and how to scale.
+
+This topic provides information for scaling the metrics components.
+
+[[cluster-metrics-scaling-openshift-metrics-pods]]
+== Scaling {product-title} Metrics Pods
+
+One set of metrics pods (Cassandra/Hawkular/Heapster) is able to monitor at
+least 10,000 pods.
```

Comment:
I do not know where it was lost, but @mffiedler and @ekuric addressed this in https://github.com/ekuric/openshift-docs/blob/c56b42891a18f6a1c9483b1d8cfec4135c88c331/admin_guide/metrics_recommendations.adoc @jeremyeder